docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -342,3 +342,21 @@ _view=sumologic_slo_output sloId="<your-SLO-ID>"
 ```
 
 These log messages will be delayed by one hour, as the system ensures consistency to account for ingest delay of source telemetry.
+
+
+### Create an SLO Lookup Table
+
+If you have an existing [Lookup Table](/docs/search/lookup-tables) you can enrich it detailed SLO data (such as sloName, signalType, sliQueryType, sloDescription) using the `_view=sumologic_slo_output` snippet.
+
+As an example, say you had a SLO [dashboard](/docs/dashboards-new) and wanted to see error budget burndown from several of your apps and services combined.<br/><img src={useBaseUrl('img/observability/percent-error-remain.png')} alt="percent-error-remain" width="450"/>
+
+You would need to create a custom graphic that combines multiple SLOs from multiple services:
+
+1. Go to **Manage Data** > **Monitoring** > **SLO**.
+1. Click on any SLO line item.
+1. Hover over the **Percentage budget remaining** panel, then click the three-dot icon > **Open in Log Search**.<br/><img src={useBaseUrl('img/observability/open-in-logsearch.png')} alt="open-in-logsearch" width="150"/>
+1. In the search field, enter the following snippet. This will join data from multiple sources for your lookup table.
+  ```
+  ??
+  ```
```

Comment:
Need an example here

Reply:
Sharing some screen shots:
<img width="773" alt="Screenshot 2023-02-14 at 11 08 17 AM" src="https://user-images.githubusercontent.com/15211243/218649247-d31b1f6b-8375-4f63-8521-0a8a4ab9ff18.png
<img width="1481" alt="Screenshot 2023-02-14 at 11 08 33 AM" src="https://user-images.githubusercontent.com/15211243/218649268-e47c0385-ff16-42de-a136-e53be7a54ba5.png">
">

Reply:
<img width="773" alt="Screenshot 2023-02-14 at 11 08 17 AM" src="https://user-images.githubusercontent.com/15211243/218649348-19e8b6e0-75e6-4e3b-bf38-f39fa7d89dde.png">

Reply:
<img width="1481" alt="Screenshot 2023-02-14 at 11 08 33 AM" src="https://user-images.githubusercontent.com/15211243/218649394-83a3da0b-b300-439e-a448-5102350d9ec4.png">

Reply:
`SLO Summary from Lookup` - can be accessed on `us2Data`

Reply:
thank you!